compiler-rt/test/tysan/violation-pr47137.c

```diff
@@ -23,8 +24,8 @@ void f(int m) {
   }
 
   // CHECK:      TypeSanitizer: type-aliasing-violation on address
-  // CHECK-NEXT: READ of size 2 at {{.+}} with type short accesses an existing object of type long long
-  // CHECK-NEXT:    in f violation-pr47137.c:30
+  // CHECK-NEXT: READ of size 2 at {{.+}} with type short accesses an existing object of type {{(long)+}}
```

Comment:
IMO having an optional (with `?`) long is preferable to having an unbound number of longs.

Reply:
Good catch, will change that 👍